deploy/charts/ray/values.yaml

```diff
@@ -71,9 +71,5 @@ namespacedOperator: false
 # operatorNamepsace - If using a cluster-scoped operator (namespacedOperator: false), set the namespace
 # in which to launch the operator.
 operatorNamespace: default
-# operatorImage - For best stability, it's currently recommended to use the nightly ray image in the operator Deployment.
-# For an official ray release version, use `rayproject/ray:1.x.y`.
```

Comment:
Do we still have these instructions somewhere in the docs? They seem useful :)

Reply:
Images are documented in the installation docs, but doesn't hurt to be redundant.